### ISSUE TITLE
Override captcha settings per instance / local

### ISSUE THREAD

**Issue by kszymukowicz** [studiomitte/friendlycaptcha-typo3#11]
hi!

Is there a good way to override captcha settings per instance or on local development?
I know we have "baseVariants" but I do not suppose it is supported for every property.

**Comment by 80Quattro** [comment]
Hi! 
@woemar We also use it this way in our projects. It works fine.
@kryslin thanks for the PR, but I don’t see a need to implement it.

**Comment by kszymukowicz** [comment]
@80Quattro 

Could you share how do you update the values at local instance then? 
I suppose you have some testing keys for FC for local development?
Then when, for example this FC_SITE_KEY, FC_SECRET is changed, you ask all developers at your team to change it locally?

**Comment by 80Quattro** [comment]
@kszymukowicz yes, exactly. We use two key pairs: one for production, stored in the .env file on the server, and one for development, which developers keep in their local .env files. Since the keys don’t change often, updating them locally isn’t an issue for us. 
They can even be the same test keys for all projects. They are only used to verify that the captcha is correctly configured in the form and to test whether the form itself works properly.
